WooCommerce/Classes/ViewRelated/Upgrades/StorePlanSynchronizer.swift

```diff
@@ -71,8 +71,9 @@ final class StorePlanSynchronizer: ObservableObject {
             return
         }
 
-        // If the site is not a WPCom store set the state to `.unavailable`
-        guard site.isWordPressComStore else {
+        // If the site is not a WPCom store and has never run a trial WooExpress plan,
```

Comment:
Considering that we are checking for OR below,
```suggestion
        // If the site is not a WPCom store or has never run a trial WooExpress plan,
```

Reply:
It should be `and` because this is a `guard` check, not `if` 😄

Reply:
Oh, right. 👍 😇